UQ-eLIPSE/crucible
# Bugfix/TEAM24-45

Refactored SSO functionality and applied to all `isAdmin` calls.

To test:
1. Go to https://vetshub-uat.uqcloud.net/
2. Click on Learning Resources.
3. Create a test collection
4. Create a quiz resource inside that collection
5. Check if it is visible.
6. Set the collection to be hidden
7. Create another quiz resource and see if that's been added.
